config/class-site-details-index.php

```diff
@@ -111,43 +111,81 @@ public function get_plugin_info() {
 			}
 
 			$plugin_info[] = array(
-				'path'         => $plugin_path,
-				'name'         => $plugin_data['Name'],
-				'wporg_slug'   => $wporg_slugs[ $plugin_path ] ?? null,
-				'version'      => $plugin_data['Version'],
-				'active'       => null !== $activated_by,
-				'activated_by' => $activated_by,
+				'path'          => $plugin_path,
+				'name'          => $plugin_data['Name'],
+				'version'       => $plugin_data['Version'],
+				'active'        => null !== $activated_by,
+				'activated_by'  => $activated_by,
+				'wporg_slug'    => isset( $update_data[ $plugin_path ] ) ? $update_data[ $plugin_path ]['slug'] : null, // legacy, can be later removed.
```

Comment:
If it's legacy now, can it be removed now, rather than "later" that may never come?